connector/mqtt/vernemq/v2k-brigde/bin/scripts_tls/signCert.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#########################################################
+#########################################################
+
+BASE_DIR=${BASE_DIR:-"/v2k_bridge"}
+
+. "${BASE_DIR}"/bin/scripts_tls/_initVariables.sh
+
+
+. "${BASE_DIR}"/bin/scripts_tls/saveFormattedCRT.sh
+
+echo "Signing cert for entity ${certCname} in ${certCAName} : ${certEjbcaApiUrl}/sign/${certCname}/pkcs10 "
+csrContent=$(<  "${certDir}"/"${certCsrFile}"  sed '1,1d;$ d' | tr -d '\r\n')
```

Comment:
Would the command below generate the same result? if so, it would be better to read because the `< `sign at the beginning of the expression was kind of weird, but if this was the purpose, then ignore my comment

`csrContent=$(cat "${certDir}/${certCsrFile}" | sed '1,1d;$ d' | tr -d '\r\n')`

Reply:
It's a recommendation from shellcheck =)